index.html

```diff
@@ -87,97 +97,101 @@ <h5>CANOPY</h5>
                     <li><a href="#"><img src="image-source/medium.svg" alt="Vector-1"></a></li>
                     <li><a href="https://github.com/Moses-chibuike"><img src="image-source/github.svg" alt="github"></a></li>
                     <li><a href="#"><img src="image-source/angellist.svg" alt="Vector"></a></li>
                 </ul>
-
-            <div class="section-2-description">
-                <button class="btn" type="button"><a href="#">Get my resume</a></button>
+                <div class="section-two-desc">
+                    <button class="btn" type="button">Get my resume</button>
+                </div>
             </div>
 
-            <ul class="skeleton">
-                <li>languages</li>
-                <div class="vector"><img src="image-source/Icon - Chevron - Down - Larger.png" alt=""></div>
-            </ul>
-
-            <ul class="languages">
-                <li class="language">
-                    <div class="languages-logo"><img src="image-source/Ellipse 1.png" alt="javascript"></div>
-                    <p>javascript</p>
-                </li>
-                <li class="language">
-                    <div class="languages-logo"><img src="image-source/Ellipse 2.png" alt="javascript"></div>
-                    <p>HTML</p>
-                </li>
-                <li class="language">
-                    <div class="languages-logo"><img src="image-source/Ellipse 3.png" alt="javascript"></div>
-                    <p>CSS</p>
-                </li>
-            </ul>
-
-            <ul class="skeleton">
-                <li>Frameportfolio-works</li>
-                <div class="vector"><img src="image-source/Icon - Chevron - Right - Larger.png" alt=""></div>
-            </ul>
-
-            <ul class="skeleton">
-                <li>Skills</li>
-                <div class="vector"><img src="image-source/Icon - Chevron - Right - Larger.png" alt=""></div>
-            </ul>
+            <div class="two">
+                <ul class="framework initial">
+                    <li>languages</li>
+                    <div class="vector"><img src="image-source/Icon - Chevron - Down - Larger.png" alt=""></div>
+                </ul>
+    
+                <ul class="languages">
+                    <li class="language">
+                        <div class="languages-logo"><img src="image-source/Ellipse 1.png" alt="javascript"></div>
+                        <p>javascript</p>
+                    </li>
+                    <li class="language">
+                        <div class="languages-logo"><img src="image-source/Ellipse 2.png" alt="javascript"></div>
+                        <p>HTML</p>
+                    </li>
+                    <li class="language">
+                        <div class="languages-logo"><img src="image-source/Ellipse 3.png" alt="javascript"></div>
+                        <p>CSS</p>
+                    </li>
+                </ul>
```

Comment:
- [ ] [OPTIONAL] You have done a great job implementing the responsive languages. 👏 . But to stick to the Figma design, I suggest that you reduce the `padding` and `font-size` of the image and text. This is because it is making the `languages` column instead of the row as you reduce the screen size 😞 . 

- Your design 😎 
![Capture](https://user-images.githubusercontent.com/99394374/213836956-722c89ac-81b0-4c28-a805-b3bf86556fe2.PNG)

Reply:
Thanks for the suggestion @AlphaNtihinduka, will do my best work on it.